corgi/collectors/brew.py

```diff
@@ -730,14 +730,27 @@ def get_maven_build_data(build_info: dict, build_type_info: dict) -> dict:
         return component
 
     @classmethod
-    def _extract_advisory_ids(cls, build_tags: list) -> list:
+    def _extract_advisory_ids(cls, build_tags: list[str]) -> list[str]:
+        """From a Brew build's list of tags, return any errata IDs with -released stripped"""
         advisory_ids = set()
         for tag in build_tags:
             match = ADVISORY_REGEX.match(tag)
             if match:
                 advisory_ids.add(match.group())
         return list(advisory_ids)
 
+    @staticmethod
+    def _parse_advisory_ids(errata_tags: list[str]) -> list[str]:
+        """From a Brew build's list of Errata tags, return tags with released (4-digit) IDs"""
+        # released errata always have 4-digit IDs, e.g. RHBA-2023:1234
+        # dropped errata always have 5-digit IDs, e.g. RHEA-2023:12345
+        # pending errata always have 6-digit IDs, e.g. RHSA-2023:123456
```

Comment:
You can just group it into one: "unreleased errata have 5-or-more-digit IDs". Dropped is just a different state of an unreleased erratum, same as QE for example. The only difference is that the QE state may result in an erratum being released whereas dropped means it will always remain unreleased.